src/Lead.jsx

```diff
@@ -1,19 +1,19 @@
 import React from 'react';
 import HornedBeasts from './HornedBeasts';
-import data from './data.json';
+// import data from './data.json';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import Container from 'react-bootstrap/Container';
 
 class Lead extends React.Component {
     render() {
         let hornedBeastComponents = [];
-        data.forEach((beasts) => {
+        this.props.beast.forEach((beast) => {
```

Comment:
this.props.beast is undefined, should be this.props.beasts like you have it in app